bin/golden.go

```diff
@@ -221,17 +224,22 @@ func doGolden() {
 	logger := logging.GetLogger(config_obj, &logging.ToolComponent)
 	logger.Info("Starting golden file test.")
 
-	globs, err := filepath.Glob(fmt.Sprintf(
-		"%s*.in.yaml", *golden_command_prefix))
+	globs, err := filepath.Glob(fmt.Sprintf("%s/**/*.in.yaml", *golden_command_prefix))
```

Comment:
I guess this is not used any more?

Reply:
oops, sorry, i had to do `.walk` since a double star glob wasn't working